pom.xml

```diff
@@ -80,6 +89,13 @@
 		</dependency>
 
 		<dependency>
```

Comment:
I'm using guava to get permutations for a property-based test that I didn't want to include in this pull request.

should fit into the next one